AppServer/google/appengine/api/memcache/memcache_stub.py

```diff
@@ -322,6 +335,7 @@ def _Dynamic_Increment(self, request, response):
           memcache_service_pb.MemcacheServiceError.UNSPECIFIED_ERROR)
     response.set_new_value(new_value)
 
+  @apiproxy_stub.Synchronized
```

Comment:
Should _Dynamic_Increment also have @apiproxy_stub.Synchronized?

Reply:
It looks like the inner function (_internal_increment) has it, and that's the only portion that has a side effect. The rest of the function is just reading the request and preparing the response.

I should mention for context that this particular part of the code is not used in AppScale. It's useful as a reference for implementing the API calls, but it does not share state between AppServer processes. Our implementation is in the memcache_distributed module.